eng/versioning/version_client.txt

```diff
@@ -66,6 +66,8 @@ com.microsoft.azure:azure-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
 com.microsoft.azure:azure-spring-boot;2.3.3-beta.1;2.3.3-beta.1
 com.microsoft.azure:spring-cloud-azure-appconfiguration-config-web;1.2.8-beta.1;1.2.8-beta.1
 com.microsoft.azure:spring-cloud-azure-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-azure-context;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
@saragluna all of these spring-cloud-* artifacts should be azure-spring-cloud-* according to the naming convention.

Reply:
Issue filed: https://github.com/Azure/azure-sdk-for-java/issues/13523